src/components/organisms/NavigatorPane/NavigatorPane.styled.tsx

```diff
@@ -36,10 +36,24 @@ export const TitleBarRightButtons = styled.div`
   padding-right: 16px;
 `;
 
-export const PlusButton = styled(Button)``;
-
 export const FilterButton = styled(Button)``;
 
 export const FiltersNumber = styled.div`
   margin-left: 5px;
 `;
+
+interface PlusItem extends ButtonProps {
+  highlighted?: boolean;
+}
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+export const PlusButton = styled(({highlighted, ...rest}: PlusItem) => <Button {...rest} />)`
```

Comment:
here you don't have to create a function on every line
you can just do something like this:
```
  ${({highlighted}) => `
      border-radius: ${highlighted ? '100%' : 'inherit'} !important;
      color: ${highlighted ? Colors.whitePure : Colors.blue6} !important;
  `};
```
and the same for `&:after`